configs/templates/ufs-utils/spack.yaml

```diff
@@ -0,0 +1,43 @@
+spack:
+  concretizer:
+    unify: when_possible
+
+  config:
+    install_tree:
+      root: $env/install
+  modules:
+    default:
+      roots:
+        lmod: $env/install/modulefiles
+        tcl: $env/install/modulefiles
+
+  view: false
+  include: []
+
+  specs:
+    - ufs-utils-env~debug  
+
+  view: false
```

Comment:
The CI unit tests found an error:
```
FAILED tests/test_stack_create.py::test_apps[ufs-utils] - spack.util.spack_yaml.SpackYAMLError: error parsing YAML: near /home/runner/work/spack-stack/spack-stack/configs/templates/ufs-utils/spack.yaml, 1, 2: found duplicate key "view" with value "False" (original value: "False")
```

Please remove lines 20 and 21 (just for the sake of not having unnecessary blank lines), hopefully that will do.

Reply:
@climbfuji I just make the correction. Please see my not in the commit.